Mlem/API/APIRequest.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 // MARK: - APIRequest
 
-enum APIRequestError: Error {
+enum APIRequestError: Error 
```

Comment:
Unintentional?

Reply:
Yes! Trying to make sure Xcode error reporting works.